libsolidity/formal/SymbolicState.cpp

```diff
@@ -149,6 +153,120 @@ smtutil::Expression SymbolicState::txConstraints(FunctionDefinition const& _func
 	return conj;
 }
 
+void SymbolicState::buildABIFunctions(set<FunctionCall const*> const& _abiFunctions)
+{
+	map<string, SortPointer> functions;
+
+	for (auto const* funCall: _abiFunctions)
```

Comment:
The order here really doesn't matter?

Reply:
Which order?

Reply:
The order of visiting the calls? I don't think so

Reply:
Ah, yes, it's only passed down as name-sorted map in the end, should be fine.